packages/test-app/src/app/app.component.html

```diff
@@ -213,4 +213,12 @@
   your nationality, you’ll have to send copies of identity documents through the post.
 </admiralty-read-more>
 <br />
+<div style="display: flex">
+  <admiralty-select style="margin-right: 10px; width: 250px" [label]="'Vertical datum *'">
```

Comment:
very minor tweak but lets not encourage people to do this 
```suggestion
  <admiralty-select style="margin-right: 10px; width: 250px" label="Vertical datum *">
```